app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+
+  def index
+    @users = User.all
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to @user, notice: "You're in!"
+    else
+      render action: 'sign_up'
+    end
+  end
+
+private
+  def user_params
+    params
+      .require(:user)
+      .permit(:first_name, :last_name, :email, :password, :password_confirmation)
```

Comment:
Line is too long. [81/80]<br>Place the . on the previous line, together with the method call receiver.